tasks/coverage.py

```diff
@@ -176,12 +177,51 @@ def upload_coverage_to_s3(ctx: Context):
     print(color_message(f'Successfully removed the local {COV_ARCHIVE_NAME}', Color.GREEN))
 
 
-def apply_missing_coverage(ctx: Context, from_commit_sha: str, debug: bool = False):
+def _get_dev_coverage_files(dev_cov_lines: str) -> set[str]:
+    browsed_dev_files = set()
+    for line in dev_cov_lines:
+        file_path = line.split(':')[0]
+        browsed_dev_files.add(file_path)
+    return browsed_dev_files
+
+
+def _merge_dev_in_main_coverage(main_cov_file: str, dev_cov_file: str) -> None:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Can you add a tiny example of what these files look like?